doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Usage & Billing"
+description: "Provides an overview of various factors that can affect cost."
+menu:
+  main:
+    parent: "administration"
+    weight: 10
+---
+
+From the [console](https://console.materialize.com/) (`Admin` > `Usage &
+Billing`), administrators can access their invoice. The invoice provides
+Compute and Storage usage and cost information.
+
+## Cost Factors
```

Comment:
```suggestion
## Cost factors
```

Reply:
I wish there were a more crisp description of what costs money, before we get into specifics (_e.g._ in Materialize you pay for compute and storage). Also, it'd be great to explain that cluster sizes actually hint at their inherent cost (`cc` stands for centicredits), as explained in [this blogpost](https://materialize.com/blog/new-cluster-sizes/).